setup.py

```diff
@@ -30,24 +31,26 @@
 }
 
 extras_require['dev'] = (
-    extras_require['dev'] +
-    extras_require['test'] +
-    extras_require['lint'] +
+    extras_require['dev'] +  # noqa: W504
+    extras_require['test'] +  # noqa: W504
+    extras_require['lint'] +  # noqa: W504
     extras_require['doc']
 )
 
+
 with open('./README.md') as readme:
     long_description = readme.read()
 
+
 setup(
     name='eth-typing',
     # *IMPORTANT*: Don't manually change the version here. Use `make bump`, as described in readme
     version='2.2.0',
     description="""eth-typing: Common type annotations for ethereum python packages""",
     long_description=long_description,
     long_description_content_type='text/markdown',
-    author='The eth-typing contributors',
-    author_email='eth-typing@ethereum.org',
+    author='The Ethereum Foundation',
```

Comment:
Same here, not a strong stance, just a slight preference of using *The `<project-name>` authors*